timemachine/cpp/src/local_md_potentials.cu

```diff
@@ -55,6 +61,13 @@ LocalMDPotentials::LocalMDPotentials(const int N, const std::vector<std::shared_
     // Add the restraint potential and ixn group potential
     all_potentials_.push_back(bound_restraint_);
```

Comment:
I would probably do a quick rename here:

`bound_restraint_` -> `bound_free_restraint_`
`reference_bound_restraint_` -> `bound_frozen_restraint_`

Reply:
Renamed in https://github.com/proteneer/timemachine/pull/1014/commits/19c74bbd25d29ccc608a25086f44b51dd404fdfb